### DISCUSSION TITLE
geometry shader demo

### DISCUSSION THREAD

**Discussion by JackyLucifer** [vsg-dev/VulkanSceneGraph#947] [Q&A] [answered]
Can you provide an example of a geometry shader

**Comment by robertosfield** [comment]
Asking others to do work for you isn't a Q&A topic. Please realize members of the VSG community are busy and we can't freely respond to requests.

**Comment by andesengineering** [comment]
Writing geometry shaders with VSG is a simple addition of code for the
geometry stage.  do be aware, however, that you need to enable this:

windowTraits->deviceFeatures->get().geometryShader = VK_TRUE;

-don

**Reply by JackyLucifer** [reply, marked as answer]
最新版本要在设置 meshshader true之后
再设置windowTraits->deviceFeatures->get().geometryShader = VK_TRUE;
否则无效
